feed.xml

```diff
@@ -20,6 +20,7 @@ layout: compress
     <itunes:explicit>{{ site.feed.explicit }}</itunes:explicit>
     <itunes:image href="{{ urlbase }}{{ site.feed.image }}" />
     <itunes:category text="{{ site.feed.category }}"/>
+    <copyright>Copyright 2020-2021 Remote Hack Space</copyright>
```

Comment:
could this be defined in jekyll config instead of a string?

Reply:
Oh, absolutely! That's a **way** better place to define it.

Reply:
Could probably make it generate this dynamically using the first and last episode dates to get the years, and pulling the "Remote Hack" bit straight out of the config

Reply:
Max off the top rope with an outstanding suggestion!